README.md

```diff
@@ -471,6 +471,9 @@ router
     if (context.params && context.params.id && books.has(context.params.id)) {
       context.response.body = books.get(context.params.id);
     }
+  })
+  .post("/", async (context) => {
```

Comment:
Why are you adding code here?

Reply:
I want to make sure people know how to write multiple methods and handle requests